src/Statistics/EmployeeParticipationShare.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Statistics;
+
+use VSV\GVQ_API\Company\ValueObjects\PositiveNumber;
+
+class EmployeeParticipationShare
```

Comment:
Misschien als naam `EmployeeParticipationRatio`, want het is de verhouding tussen deelnames en totaal aantal mogelijke deelnemers.